internal/pkg/template/templates/overrides/cdk/stack.ts

```diff
@@ -4,14 +4,19 @@ import * as path from 'path';
 import { {{$resourceType.ImportName}} as {{$resourceType.ImportShortRename}} } from 'aws-cdk-lib';
 {{- end }}
 
+interface TransformedStackProps extends cdk.StackProps {
+    readonly appName?: string;
+    readonly envName?: string;
+}
+
 export class TransformedStack extends cdk.Stack {
-    constructor (scope: cdk.App, id: string, props?: cdk.StackProps) {
+    constructor (scope: cdk.App, id: string, props: TransformedStackProps) {
```

Comment:
if `appName` and `envName` are optional, why do we make `props` required?

Reply:
good question 😂 changed to `appName` and `envName` being required